freezeyt/hooks.py

```diff
@@ -39,4 +56,13 @@ def done_task_count(self):
         # Import TaskStatus here to avoid a circular import
         # (since freezer imports hooks)
         from freezeyt.freezer import TaskStatus
-        return len(self._freezer.task_queues[TaskStatus.DONE])
+        return (
+            len(self._freezer.task_queues[TaskStatus.FAILED])
```

Comment:
```suggestion
            self.failed_task_count
```
Does it make any improvement?